src/features/register_feature_options.js

```diff
@@ -130,6 +130,13 @@ registerFeature({
   category: "Editing",
 });
 
+registerFeature({
+  name: "Clipboard and Notes",
+  id: "clipboardAndNotes",
+  description: "Keep useful things for pasting in the clipboard and useful notes in the notes.",
+  category: "Global",
+});
```

Comment:
When my change is merged you will want to set the defaultValue here

Reply:
OK. Thanks.